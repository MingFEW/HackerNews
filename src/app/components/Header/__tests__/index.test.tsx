import { createRenderer } from 'react-test-renderer/shallow'

import { Header } from '../index'

const shallowRenderer = createRenderer()

describe('<Header />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<Header />)
    const renderedOutput = shallowRenderer.getRenderOutput()
    expect(renderedOutput).toMatchSnapshot()
  })
})
