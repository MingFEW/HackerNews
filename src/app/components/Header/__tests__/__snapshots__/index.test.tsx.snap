// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render and match the snapshot 1`] = `
<div
  tw="fixed w-screen backdrop-blur z-[2] border-b border-solid border-white/20"
>
  <div
    className="header"
    tw="px-6 py-4 mx-auto md:max-w-[640px] md:px-0 lg:max-w-[768px]"
  >
    <h1
      tw="text-white text-xl font-bold"
    >
      Hacker Top Stories
    </h1>
    <p
      tw="text-textGrey text-xs"
    >
      Made by MingFEW
    </p>
  </div>
</div>
`;
